OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -676,6 +676,17 @@ static void HideChildWidget(Widget parent, string widgetId)
 			if (widget != null)
 				widget.IsVisible = () => false;
 		}
+
+		public static string SanitizePlayerName(Session.Client client)
+		{
+			if (Game.Settings.Game.SanitizeMutedPlayerNames && TextNotificationsManager.MutedPlayers[client.Index])
```

Comment:
MutedPlayer's is empty inside ingame lobby so this will crash. Though we I recon we do want to add muting to the lobby as well, it was requested on discord afaik